packages/edit-post/src/components/header/plugin-more-menu-item/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose';
+import { MenuItem } from '@wordpress/components';
+import { withPluginContext } from '@wordpress/plugins';
+
+/**
+ * Internal dependencies
+ */
+import PluginsMoreMenuGroup from '../plugins-more-menu-group';
+
+const PluginMoreMenuItem = ( { url, onClick = noop, ...props } ) => (
+	<PluginsMoreMenuGroup>
+		{ ( fillProps ) => (
+			<MenuItem
+				{ ...props }
+				href={ url }
```

Comment:
I think this prop should be just be `href` and rely on `{...props}` for consistency.

Reply:
`href` is very specific to anchor, but I get your point. I will update.